recipes-bsp/device-tree/files/tepzu.dts

```diff
@@ -0,0 +1,370 @@
+					pins =	"gpio0", "gpio1", "gpio2", "gpio3",
+						"gpio4", "gpio5", "gpio6", "gpio7";
+					bias-pull-up;
+				};
+
+				hog-hreset {
+					gpio-hog;
+					gpios = <3 0>;
+					output-low;
+					line-name = "HRESET";
+				};
+			};
+
+			/* Also connected to PCIe switch */
+		};
+
+		/* FMC */
+		i2cmux_1: i2c@1 {
+			#address-cells = <1>;
+			#size-cells = <0>;
+			reg = <1>;
+		};
+
+		/* SYZYGY */
+		i2cmux_2: i2c@2 {
+			#address-cells = <1>;
+			#size-cells = <0>;
+			reg = <2>;
+		};
+
+		/* FTDI, USB, DP */
+		i2cmux_3: i2c@3 {
+			#address-cells = <1>;
+			#size-cells = <0>;
+			reg = <3>;
+		};
+	};
+};
+
+/* EEPROM on carrier is connected to bus 0 */
+&i2c1 {
+	carrier_eeprom: m24c32@50 {
+		compatible = "atmel,24c32";
+		reg = <0x50>;
+		pagesize = <32>;
+		vcc-supply = <&reg_1v8_miami>;
+		#address-cells = <1>;
+		#size-cells = <1>;
+	};
+};
+
+&uart1 {
+	status = "disabled";
+};
+
+&usb0 {
+	status = "okay";
+	/* see: https://forums.xilinx.com/t5/Embedded-Linux/Zynqmp-USB2-0/td-p/790522/page/2 */
+	/delete-property/ clocks;
+	/delete-property/ clock-names;
+	clocks = <&zynqmp_clk USB0_BUS_REF>;
+	clock-names = "bus_clk";
+};
+
+&dwc3_0 {
+	status = "okay";
+	dr_mode = "otg"; /* "host", "peripheral" or "otg" */
+	maximum-speed = "high-speed"; /* super-speed not available on carrier */
+	/* see: https://forums.xilinx.com/t5/Embedded-Linux/Zynqmp-USB2-0/td-p/790522/page/2 */
+	snps,dis_u2_susphy_quirk;
+	snps,dis_u3_susphy_quirk;
+	/delete-property/ phy-names;
+	/delete-property/ phys;
+	/delete-property/ snps,usb3_lpm_capable;
+	vbus-supply = <&reg_usb0_vbus>;
+	connector {
+		compatible = "usb-b-connector";
+		label = "micro-USB-otg";
+		type = "micro";
+	};
+};
+
+&usb1 {
+	status = "okay";
+	/*
+	 * Using Xilinx' borked implementation to add a reset for the HUB,
+	 * otherwise SuperSpeed doesn't work.
+	 */
+	reset-gpios = <&tep_gpioex10 5 GPIO_ACTIVE_LOW>;
+};
+
+&dwc3_1 {
+	status = "okay";
+	dr_mode = "host";
+	/* refclk0 from clock synth */
+	assigned-clocks = <&si5345 0 3>;
+	assigned-clock-rates = <100000000>;
+	phy-names = "usb3-phy";
+	phys = <&psgtr 3 PHY_TYPE_USB3 1 0>;
+	maximum-speed = "super-speed";
+	snps,usb3_lpm_capable;
+};
+
+&pcie {
+	status = "okay";
+	phys = <&psgtr 0 PHY_TYPE_PCIE 0 0>;
+	/* refclk0 from clock synth */
+	assigned-clocks      = <&si5345 0 3>;
+	assigned-clock-rates =   <100000000>;
+};
+
+/* Displayport - HP and AUX signals through EMIO, enabled when PL probes */
+&zynqmp_dpsub {
+	phy-names = "dp-phy0";
+	phys = <&psgtr 1 PHY_TYPE_DP 0 3>;
+	xlnx,max-lanes = <1>;
+	assigned-clocks = <&si5345 0 8>;
+	assigned-clock-rates = <27000000>;
+	assigned-clock-parents = <&si5345 1 1>;
+};
+&zynqmp_dpdma {
+	status = "okay";
+};
+
+&si5345 {
+	assigned-clock-parents = <0>, <0>, <0>, <0>, <0>,
+				<&si5345 1 0>, /* out 0 */
+				<&si5345 1 0>,
+				<&si5345 1 0>,
+				<&si5345 1 0>,
+				<&si5345 1 0>,
+				<&si5345 1 0>, /* out 5 */
+				<&si5345 1 0>,
+				<&si5345 1 0>,
+				<&si5345 1 1>,
```

Comment:
What does parameter 2 = '1' here refer to? I just see that it is different from the rest.

Reply:
Sets up the clock parent to be "1" instead of "0" (so it no longer shares PLL 0  but gets its own private PLL 1)